docroot/modules/custom/nvli_custom_search/src/Plugin/Block/NewSearchBlock.php

```diff
@@ -25,7 +25,7 @@ class NewSearchBlock extends BlockBase {
    * {@inheritdoc}
    */
   protected function blockAccess(AccountInterface $account) {
```

Comment:
@dishabhadra What is this block intended for? The name NewSearchBlock doesn't give any insight about it.